miniapp/miniapp_band_to_tridiag.cpp

```diff
@@ -8,10 +8,12 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 
+#include <cstdlib>
 #include <iostream>
 #include <limits>
 
 #include <pika/init.hpp>
+#include <pika/init_runtime/scoped_finalize.hpp>
```

Comment:
```suggestion
```
and the same in the other miniapps of course.

We added the very minimum of documentation in 0.5.0: https://github.com/pika-org/pika#documentation :sweat_smile: I know there's a long way to go...

Reply:
Yep: https://github.com/pika-org/pika/blob/977f4533ccb9f773af004e9f58b7802d250b4b70/libs/pika/init_runtime/include/pika/init.hpp#L10.